### PR TITLE
Box the Simulator

### PR DESCRIPTION
**Pull request by BloomyInDev**

### CODE REVIEW

File: src/routes/Viewer.vue [hunk 1]
```diff
@@ -48,6 +63,11 @@ export default defineComponent({
     }
     this.globalStore.progress = false;
   },
+  methods: {
+    launchSimul() {
```
Comment (Yaya-Cout) [inline src/routes/Viewer.vue]: Don't use abbreviations
```suggestion
    launchSimulator() {
```

File: src/routes/Viewer.vue [hunk 2]
```diff
@@ -36,6 +50,7 @@ export default defineComponent({
       api: useAPIStore().api,
       globalStore: useGlobalStore(),
       uuid: this.$route.params.uuid as string,
+      startSimul: false
```
Comment (Yaya-Cout) [inline src/routes/Viewer.vue]: Same
```suggestion
      startSimulator: false
```

File: src/locales/fr.json [hunk 3]
```diff
@@ -199,6 +199,8 @@
     "modified-on": "Modifié pour la dernière fois le {date}",
     "upload": "Envoyer vers la calculatrice",
     "version": "Version {version}",
-    "views": "{count} vues"
+    "views": "{count} vues",
+    "simulator-launch":"Lancer le simulateur",
+    "simulator-stop":"Stopper le simulateur"
```
Comment (Yaya-Cout) [inline src/locales/fr.json]:
```suggestion
    "simulator-launch": "Lancer le simulateur",
    "simulator-stop": "Stopper le simulateur"
```

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]

File: src/routes/Viewer.vue (hunk 1 above)
Comment (BloomyInDev) [reply src/routes/Viewer.vue]: Sry

**Review by Yaya-Cout — CHANGES_REQUESTED** [review]
You have to rename the other variables to match the rename

**Pull request closed without merging**